Formula/ace.rb

```diff
@@ -1,8 +1,8 @@
 class Ace < Formula
   desc "ADAPTIVE Communication Environment: OO network programming in C++"
-  homepage "https://www.dre.vanderbilt.edu/~schmidt/ACE.html"
-  url "https://github.com/DOCGroup/ACE_TAO/releases/download/ACE%2BTAO-6_5_12/ACE-6.5.12.tar.bz2"
-  sha256 "ccd94fa45df1e8bb1c901d02c0a64c1626497e5eeda2f057fcf0a1578dae2148"
+  homepage "https://github.com/DOCGroup/ACE_TAO"
```

Comment:
Do we really want to update homepage? 
GitHub repo mentions https://www.dre.vanderbilt.edu/~schmidt/TAO.html as a homepage

Reply:
Reverted that